service/src/main/kotlin/fi/espoo/evaka/incomestatement/IncomeStatementController.kt

```diff
@@ -149,3 +152,12 @@ class IncomeStatementController(
         }
     }
 }
+
+data class SearchIncomeStatementsRequest(
+    val page: Int = 1,
+    val pageSize: Int = 50,
+    val areas: List<String>? = emptyList(),
+    val providerTypes: List<ProviderType>? = emptyList(),
+    val sentStartDate: String? = null,
+    val sentEndDate: String? = null,
```

Comment:
You can use the `LocalDate` type here and remove the parsing code from the route handler.